scripts/create-cluster.sh

```diff
@@ -37,5 +37,6 @@ eksctl create cluster \
 --node-type m5.xlarge \
 --nodes ${DESIRED_NODE} \
 --nodes-min ${MIN_NODE} \
---nodes-max ${MAX_NODE}
+--nodes-max ${MAX_NODE} \
+--managed
```

Comment:
@PatrickXYS What do you think about re-using kubeflow/testing scripts to deploy EKS cluster, like I did here: https://github.com/kubeflow/katib/blob/de3c82b548022a17ffd79c1fccef978d4544b60c/test/workflows/components/workflows-v1beta1.libsonnet#L372-L374?
I believe, for PyTorch operator we also do checkout, so we can re-use /testing scripts.

Reply:
Sounds good, I think we should enforce WG reuse two common scripts (Create / Delete EKS cluster), otherwise it might make AWS resources not clean-up completely, which brings more maintenance efforts.

Reply:
This is legacy scripts. Reusing from testing image is better. Agree.  @PatrickXYS Can you make a change to testing/infra to use spot instance later? This will reduce cost a lot. Another thing to do is limit the instance types/number of instances user can use.

Reply:
1. Using Spot instance makes sense
2. Limitation on the number / instance type, this might require additional work, I need to think about how to make proper limitation